files/zh-cn/web/javascript/reference/global_objects/string/valueof/index.md

```diff
@@ -5,23 +5,23 @@ slug: Web/JavaScript/Reference/Global_Objects/String/valueOf
 
 {{JSRef}}
 
-**`valueOf()`** 方法返回 {{jsxref("String")}} 对象的原始值
+**`valueOf()`** 方法返回 {{jsxref("String")}} 对象的原始值。
```

Comment:
Let's follow the [en-US document](https://github.com/mdn/content/blob/6e78306f9674a5e6329e07eca5f2791ce3138f0c/files/en-us/web/javascript/reference/global_objects/string/valueof/index.md?plain=1#L10).

```suggestion
**`valueOf()`** 方法返回 {{jsxref("String")}} 对象的字符串值。
```

Reply:
Ok, It is string value and not primitive value.